src/cmd/manage_nsfs.js

```diff
@@ -420,13 +429,9 @@ async function validate_bucket_add_args(data) {
         console.error('Error: User data and bucket path should not be empty');
         return false;
     }
-    if (!data.path.includes(data.name.unwrap())) {
```

Comment:
why this check removed?

Reply:
there is no suck restriction - to have the bucket name in the path of the exported directory